frontend/src/metabase/query_builder/components/dataref/DatabaseTablesPane/DatabaseTablesPane.tsx

```diff
@@ -19,40 +17,39 @@ import {
 import { ModelId } from "./DatabaseTablesPane.styled";
 
 interface DatabaseTablesPaneProps {
-  show: (type: string, item: unknown) => void;
+  show: (type: string, item: any) => void;
   database: Database;
-  models: Card[];
+  searchResults: any[]; // TODO: /api/search is yet to be typed
 }
 
 const DatabaseTablesPane = ({
-  database,
   show,
-  models,
+  searchResults,
 }: DatabaseTablesPaneProps) => {
   const tables = useMemo(
-    () => database.tables.sort((a, b) => a.name.localeCompare(b.name)),
-    [database.tables],
+    () => searchResults?.filter(x => x.model === "table"),
+    [searchResults],
   );
-  const sortedModels = useMemo(
-    () => models?.sort((a, b) => a.name.localeCompare(b.name)),
-    [models],
+  const models = useMemo(
+    () => searchResults?.filter(x => x.model === "dataset"),
```

Comment:
Search results in this iteration are no longer presented in alphabetical order.

### Now

<img width="339" alt="image" src="https://user-images.githubusercontent.com/380816/193611164-2fea03f8-bc97-40da-aad3-9baea82dbdfd.png">

### Before

<img width="347" alt="image" src="https://user-images.githubusercontent.com/380816/193611203-cada32da-5478-4f64-ae9e-c71a86202700.png">

Reply:
Fixed here https://github.com/metabase/metabase/pull/25751/commits/273278be71754b4bf651d9961b406289d683b10d

Thanks for catching this!